pkgs/development/python-modules/quadprog/default.nix

```
@@ -0,0 +1,26 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, numpy
+}:
+
+buildPythonPackage rec {
+  pname = "quadprog";
+  version = "0.1.11";
```

We should limit the supported Python version. The chance is very high that nobody will test the module on unsupported releases.

```suggestion
  version = "0.1.11";
  format = "setuptools";

  disabled = pythonOlder "3.7";
```

---

Done.